packages/labs/virtualizer/src/events.ts

```diff
@@ -11,7 +11,7 @@ export class RangeChangedEvent extends Event {
   last: number;
 
   constructor(range: Range) {
-    super(RangeChangedEvent.eventName, {bubbles: true});
+    super(RangeChangedEvent.eventName, {bubbles: false});
```

Comment:
Not for this PR, but can we get code comments on the event class and on `first` and `last` to document to users and maintainers what they're for? It can just be what's in the readme here: https://github.com/lit/lit/tree/main/packages/labs/virtualizer#visibilitychanged-event